src/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -74,3 +74,7 @@ GENERATE_EXPORT_HEADER( clrcompression
 )
 
 install (TARGETS clrcompression DESTINATION .)
+
+install (FILES ${CMAKE_CURRENT_BINARY_DIR}/Debug/clrcompression.pdb DESTINATION .
+    CONFIGURATIONS Debug
```

Comment:
Cool, thanks @ianhays! I'll try out those changes on top of this PR to see if the packaging still finds things where it expects them.